### PR TITLE
Improvement regex content-range

### PR DESCRIPTION
**Pull request by ho4ho**
`*` -> 0-n
`+` -> 1-n

Regex match would:

`Content-Range: bytes -/`

Beter use `+`

### CODE REVIEW

File: lib/Sapi.php [hunk 1]
```diff
@@ -99,7 +99,7 @@ public static function sendResponse(ResponseInterface $response)
                     // If this is a partial response, flush the beginning bytes until the first position that is a multiple of the page size.
                     $contentRange = $response->getHeader('Content-Range');
                     // Matching "Content-Range: bytes 1234-5678/7890"
-                    if (null !== $contentRange && preg_match('/^bytes\s([0-9]*)-([0-9]*)\//i', $contentRange, $matches) && '' !== $matches[1]) {
+                    if (null !== $contentRange && preg_match('/^bytes\s([0-9]+)-([0-9]+)\//i', $contentRange, $matches) && '' !== $matches[1]) {
```
Comment (vfreex) [inline lib/Sapi.php]: No, `Content-Range` header is for server response, not user request.

RFC 7233 clearly defines the syntax of this header at https://tools.ietf.org/html/rfc7233#section-4.2:

>      Content-Range       = byte-content-range
>                          / other-content-range
> 
>      byte-content-range  = bytes-unit SP
>                            ( byte-range-resp / unsatisfied-range )
> 
>      byte-range-resp     = byte-range "/" ( complete-length / "*" )
>      byte-range          = first-byte-pos "-" last-byte-pos
>      unsatisfied-range   = "*/" complete-length
> 
>      complete-length     = 1*DIGIT
> 
>      other-content-range = other-range-unit SP other-range-resp
>      other-range-resp    = *CHAR

and examples

>   o  The first 500 bytes:
> 
>         Content-Range: bytes 0-499/1234
> 
>    o  The second 500 bytes:
> 
>         Content-Range: bytes 500-999/1234
> 
>    o  All except for the first 500 bytes:
> 
>         Content-Range: bytes 500-1233/1234
> 
>    o  The last 500 bytes:
> 
>         Content-Range: bytes 734-1233/1234

Unless the response is HTTP 416 (Range Not Satisfiable), the range portion is `*`.

Comment (vfreex) [reply lib/Sapi.php]: Sorry I misunderstood by only looking at the comment on #119.
Using `+` is better.

Comment (vfreex) [reply lib/Sapi.php]: you may also remove ` && '' !== $matches[1]` statement from the condition since it will not be empty.

**Comment by staabm** [comment]
thx guys

**Pull request merged**